MekHQ/src/mekhq/campaign/mission/AtBContract.java

```diff
@@ -1193,6 +1193,12 @@ protected void writeToXmlBegin(PrintWriter pw1, int indent) {
                 +"<nextWeekBattleTypeMod>"
                 +nextWeekBattleTypeMod
                 +"</nextWeekBattleTypeMod>");
+        if (parentContract != null) {
+            pw1.println(MekHqXmlUtil.indentStr(indent+1)
+                    + "<parentContractId>"
+                    + parentContract.getId()
+                    + "</parentContractId>");
```

Comment:
Use the MekHqXmlUtil.writeSimpleXmlTag instead, it handled UUID with a null check (And doesn't write otherwise)

Reply:
The ID is an integer in this case.